lib/query/utils.js

```diff
@@ -36,7 +51,7 @@ const mimeType = query => {
     return 'application/sparql-results+json';
   }
 
-  if (type === 'ask') {
+  if (type === 'ask' || type === 'update') {
     return 'text/boolean';
```

Comment:
Technically speaking, updates don't return boolean. But this probably doesn't make a difference here